Automattic/jetpack
# Move VIdeoPress v2 REST api endpoint to package

#### Changes proposed in this Pull Request:

* Move VIdeoPress v2 REST api endpoint to package

#### Other information:


#### Jetpack product discussion


pe4Cmx-40-p2

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






NOTE: I tried to initialize the package via Config::ensure, but Jetpack initialized the modules too late for that, and we only want to enable the package if the VideoPress module is active

* Add a regular video block (not the new Beta videopress block)
* Upload a video
* Click the block
* In the sidebar settings, change one of the "Video file settings" (Allow Download, for example)
* verify that there was a request to `wp-json/wpcom/v2/videopress/meta`
* verify that everything works
* Refresh the page and confirm the setting was properly saved




---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1202561463041772
